core/client/fs/src/test/java/alluxio/client/file/cache/TwoChoiceRandomEvictorTest.java

```diff
@@ -39,8 +39,6 @@ public void before() {
   public void evictGetOrder() {
     mEvictor.updateOnGet(mFirst);
     Assert.assertEquals(mFirst, mEvictor.evict());
-    mEvictor.updateOnGet(mSecond);
-    Assert.assertEquals(mSecond, mEvictor.evict());
```

Comment:
or u could do Assert.assertTrue(evictedPage.equals(mFirst) || evictedPage.equals(mSecond)); as well here?

Reply:
Yep! Thanks for your suggestion. I updated the test here.